java/ql/src/Security/CWE/CWE-502/UnsafeDeserialization.ql

```diff
@@ -18,12 +18,14 @@ import DataFlow::PathGraph
 class UnsafeDeserializationConfig extends TaintTracking::Configuration {
   UnsafeDeserializationConfig() { this = "UnsafeDeserializationConfig" }
 
-  override predicate isSource(DataFlow::Node source) { source instanceof RemoteFlowSource }
+  override predicate isSource(DataFlow::Node source) { 
+    source instanceof RemoteFlowSource 
+  }
 
   override predicate isSink(DataFlow::Node sink) { sink instanceof UnsafeDeserializationSink }
 }
 
 from DataFlow::PathNode source, DataFlow::PathNode sink, UnsafeDeserializationConfig conf
 where conf.hasFlowPath(source, sink)
 select sink.getNode().(UnsafeDeserializationSink).getMethodAccess(), source, sink,
-  "Unsafe deserialization of $@.", source.getNode(), "user input"
+ "Unsafe deserialization of $@.", source.getNode(), "user input"
```

Comment:
This seems to not be autoformatted.
You can run the autoformatter from VS Code on all files.